FieldworkAIDriver.lua

```diff
@@ -491,11 +520,13 @@ function FieldworkAIDriver:driveFieldworkUnloadOrRefill()
 			self.fieldworkUnloadOrRefillState = self.states.WAITING_FOR_UNLOAD_OR_REFILL
 		end
 	elseif self.fieldworkUnloadOrRefillState == self.states.WAITING_FOR_UNLOAD_OR_REFILL then
-		if self:allFillLevelsOk() and not self.heldForUnloadRefill then
-			self:debug('unloaded, continue working')
-			-- not full/empty anymore, maybe because Refilling to a trailer, go back to work
-			self:clearInfoText(self:getFillLevelInfoText())
-			self:changeToFieldwork()
+		if g_updateLoopIndex % 5 == 0 then --small delay, to make sure no more fillLevel change is happening
```

Comment:
isn't this rather a delay which could be anything between 0 and 4 loops? When it first comes here in loop 35, there'll be no delay.